src/renderer/components/modals/DepositWithdrawModal.vue

```diff
@@ -84,7 +86,9 @@ export default {
 
       if (cleanAmount && cleanAmount.length > 0) {
```

Comment:
This whole `cleanAmount` method seems like it could be broken up a bit better to improve testability, especially now that we are implement unit tests for components.